harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/csw/Aligner.java

```diff
@@ -427,9 +428,9 @@ private Element retrieveMetadata(String uuid) {
 
 
             try {
-                params.getValidate().validate(dataMan, context, response);
+                params.getValidate().validate(context.getBean(DataManager.class), context, response);
             } catch (Exception e) {
-                log.info("Ignoring invalid metadata with uuid " + uuid);
+                log.debug("Ignoring invalid metadata with uuid " + uuid);
```

Comment:
To check if would be better to use `warn` level.

Reply:
There is no "warn" level on that log ( org.fao.geonet.Logger )